src/bidiMapper/domains/network/networkRequest.ts

```diff
@@ -290,12 +290,12 @@ export class NetworkRequest {
           url: this.#responseReceivedEvent.response.url,
           protocol: this.#responseReceivedEvent.response.protocol ?? '',
           status:
-            this.#responseReceivedExtraInfoEvent?.statusCode ||
+            this.#responseReceivedExtraInfoEvent?.statusCode ??
             this.#responseReceivedEvent.response.status,
           statusText: this.#responseReceivedEvent.response.statusText,
           fromCache:
-            this.#responseReceivedEvent.response.fromDiskCache ||
-            this.#responseReceivedEvent.response.fromPrefetchCache ||
+            this.#responseReceivedEvent.response.fromDiskCache ??
```

Comment:
I think this change is incorrect all values are boolean if one is true we want to set the value to true.

Reply:
Will revert.